apps/services/license-api/src/app/modules/license/clients/disabilityLicense/disabilityLicenseClient.service.ts

```diff
@@ -0,0 +1,52 @@
+import { Inject, Injectable } from '@nestjs/common'
+import { GenericLicenseClient } from '../../license.types'
+import type { Logger } from '@island.is/logging'
+import { LOGGER_PROVIDER } from '@island.is/logging'
+import {
+  Pass,
+  PassDataInput,
+  Result,
+  RevokePassData,
+  SmartSolutionsApi,
+  VerifyPassData,
+} from '@island.is/clients/smartsolutions'
+
+@Injectable()
+export class DisabilityLicenseClientService implements GenericLicenseClient {
+  constructor(
+    @Inject(LOGGER_PROVIDER) private logger: Logger,
+    private smartApi: SmartSolutionsApi,
+  ) {}
+
+  pushUpdate(
+    inputData: PassDataInput,
+    nationalId: string,
+  ): Promise<Result<Pass | undefined>> {
+    this.logger.debug('in push update for Disability license')
```

Comment:
Logs should be useful and meaningful. If I saw this in the logs somewhere It would give me no value. More of this below.